fdbserver/include/fdbserver/RocksDBCheckpointUtils.actor.h

```diff
@@ -31,6 +31,20 @@
 
 #include "flow/actorcompiler.h" // has to be last include
 
+const std::string checkpointBytesSampleFileName = "metadata_bytes.sst";
```

Comment:
nit: can you move `checkpointBytesSampleFileName` to `StorageCheckpoint.h`, and move `checkpointBytesSampleTempFolder` to storageServer.actor.cpp, since it is kind of internal to ss.